build/gen_stub.php

```diff
@@ -2107,15 +2132,15 @@ public function getType(): string {
 
         if ($this->name === "param") {
             preg_match('/^\s*([\w\|\\\\\[\]<>, ]+)\s*\$\w+.*$/', $value, $matches);
-        } elseif ($this->name === "return") {
-            preg_match('/^\s*([\w\|\\\\\[\]<>, ]+)(\s+|$)/', $value, $matches);
+        } elseif ($this->name === "return" || $this->name === "var") {
+            preg_match('/^\s*([\w\|\\\\\[\]<>, ]+)\s*/', $value, $matches);
```

Comment:
What's the `\s*` at the end for?

Reply:
I think it's simply an artifact of trying to support trailing comments. I also tried to create a regex which validates the types more strictly (like how you did it), but I lacked the skills, so I like your approach :)

Reply:
I tried to make the below regex work, but I didn't manage to. That said, I'll go with my simple approach, and we can retry parsing types properly in the future.

Reply:
I've just got rid of this unnecessary change while applying the commit.